### PR TITLE
Scale graphics with window at small sizes

### PR DESCRIPTION
**Pull request by htwyford**
Resize the welcome graphics with the viewport shrinking. Addresses #191.

### CODE REVIEW

**Comment by ericawright** [comment]
a bit of dark theme css stuff accidentally snuck in....

File: addon/sidebar.css [hunk 1]
```diff
@@ -60,39 +60,53 @@ body {
   display: flex;
   flex-direction: column;
   margin-bottom: 16px;
+  width: 71vw;
+  max-width: 284px;
+  height: 52vw;
```
Comment (ericawright) [inline addon/sidebar.css]: Switch either width or height to "auto" so we don't have to think about aspect-ratio. I think one of these properties is being ignored right now anyway. And the same for below.

File: addon/sidebar.css [hunk 2]
```diff
@@ -60,39 +60,53 @@ body {
...
-  height: 174px;
+  max-width: 284px;
+  width: 100%;
+  max-height: 174px;
+  height: 100%;
   background: url(images/in-content-clouds.png);
-  background-size: 284px 174px;
+  background-size: contain;
+  background-repeat: no-repeat;
+  background-position: center;
```
Comment (ericawright) [inline addon/sidebar.css]: background: url(images/in-content-clouds.png) center no-repeat;

And below. (contain is supposed to be able to go in the one-liner too but it's misbehaving"

**Comment by ericawright** [comment]
note: we talked about why auto wasn't working, these images are difficult.
@fzzzy want to review?

**Comment by fzzzy** [comment]
I know there's a bit of dark theme css in here, but it doesn't hurt anything and when the conflicts on the css dark theme pr are resolved hopefully they just integrate these changes smoothly.

**Pull request merged**